### PR TITLE
cleanup of template

### PR DESCRIPTION
**Pull request by remco-nfi**
It seems that the template was committed with debug-state.
This cleanup reverts the plugin to its intended form.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -16,15 +16,16 @@ RUN apt-get -y update; apt-get -y install curl default-jre-headless unzip findut
 RUN python -m venv /venv
 ENV PATH="/venv/bin:$PATH"
 
+RUN curl -LO https://github.com/kaitai-io/kaitai_struct_compiler/releases/download/0.10/kaitai-struct-compiler-0.10.zip
```
Comment (remco-nfi) [inline Dockerfile:19]: putting this on top will result in more efficient images and faster build times (now the `kaitai_struct_compiler` is not downloaded/unzipped each time a python requirement was changed or a struct was updated)

File: requirements.in [hunk 2]
```diff
@@ -1,3 +1,4 @@
 hansken-extraction-plugin==0.8.1
 kaitaistruct
-json_stream
\ No newline at end of file
+json_stream
+pyyaml
```
Comment (remco-nfi) [inline requirements.in:4]: for some reason this was missing, but it is required

File: requirements.txt [hunk 3]
```diff
@@ -1,25 +1,57 @@
 certifi==2024.7.4
```
Comment (remco-nfi) [inline requirements.txt:1]: the file is now re-generated with `tox -e upgrade`.
Mose important change: `hansken_extraction_plugin==0.8.1+master.77.3b0112ba` to `hansken_extraction_plugin==0.8.1`

File: plugin.py [hunk 4]
```diff
@@ -17,13 +17,13 @@ def plugin_info(self):
         plugin_name = ''.join(letter for letter in no_space_plugin_name if letter.isalnum() or letter == '_')
         plugin_description = f'Extracts "{file_format}" files and attaches its low-level data structure as a JSON text to the trace.'
         plugin_info = PluginInfo(
-            id=PluginId(domain='communityportal.hansken.org', category='extract', name=plugin_name),
+            id=PluginId(domain='{YOUR_ORGANISATION_DOMAIN}', category='extract', name=plugin_name),
             version='1.0.0',
             description=plugin_description,
-            author=Author('Jan', 'my@email.address', 'pim organisation'),
+            author=Author('{YOUR_NAME}', '{YOUR_EMAIL_ADDRESS}', '{YOUR_ORGANISATION}'),
```
Comment (luuk-nfi) [inline plugin.py:23]: Good point, this looks a bit nicer.

**Review by luuk-nfi — APPROVED** [review]
Looks nicer and cleaner!

File: plugin.py [hunk 5]
```diff
@@ -17,13 +17,13 @@ def plugin_info(self):
...
             maturity=MaturityLevel.PROOF_OF_CONCEPT,
             webpage_url='',  # e.g. url to the code repository of your plugin
-            matcher='$data.mimeClass=archive',  # add the query for the firefli types of files your plugin should match
+            matcher='$data.fileType={FILETYPE_PROPERTY}',  # add the data stream fileType(s) of files your plugin should match
```
Comment (Marcel-NFI) [inline plugin.py:26]: Maybe also show the option of matching a file path? Maybe a comment that matching fileType is recommended (they should check the Supported Formats documentation to check if a fileType is recognized) but if not recognized that matching on file path can be used.

Comment (Marcel-NFI) [reply plugin.py:26]: The README also only mentiones that you can match on fileType. Maybe change is there (to)?

File: plugin.py (hunk 4 above)
Comment (Marcel-NFI) [reply plugin.py:23]: Nice

**Review by Marcel-NFI — APPROVED** [review]

File: plugin.py (hunk 5 above)
Comment (remco-nfi) [reply plugin.py]: Great suggestion. Improved the text a little and included a link to the documentation.

**Pull request merged**